lib/ui/painting/image_decoder.cc

```diff
@@ -282,6 +282,7 @@ static SkiaGPUObject<SkImage> UploadRasterImage(
           })
           .SetIfFalse([&result, context = io_manager->GetResourceContext(),
                        &pixmap, queue = io_manager->GetSkiaUnrefQueue()] {
+            TRACE_EVENT0("flutter", "MakeCrossContextImageFromPixmap");
```

Comment:
nit can we split tracing that does not depend on this PR into a separate PR?

We've had some issues recently where adding more tracing has run into timeouts in the devicelab, and if that happens here it'll be nice to know that's all it is and not some other change in this PR.

Also, if this has to get reverted for some reason the tracing will stay.